ansible/group_vars/all

```diff
@@ -69,7 +69,6 @@ controller:
   basePort: 10001
   heap: "{{ controller_heap | default('2g') }}"
   arguments: "{{ controller_arguments | default('') }}"
-  blackboxFraction: "{{ controller_blackbox_fraction | default(0.10) }}"
```

Comment:
Can we keep this default in for now and also define a default "managedFraction" of 1-blackboxFraction to not run at the risk of configuring overlapping pools in existent deployments that do not define a managedFraction?